mne/viz/topo.py

```diff
@@ -381,9 +380,15 @@ def _plot_topo(
 
 def _plot_topo_onpick(event, show_func):
     """Onpick callback that shows a single channel in a new figure."""
-    # make sure that the swipe gesture in OS-X doesn't open many figures
     orig_ax = event.inaxes
-    if orig_ax.figure.canvas._key in ["shift", "alt"]:
+    fig = orig_ax.figure
+
+    # If we are doing lasso select, allow it to handle the click instead.
+    if hasattr("lasso") and event.key in ["control", "ctrl+shift"]:
```

Comment:
here's the `hasattr` problem. I think `fig` is what you mean to check?

```suggestion
    if hasattr(fig, "lasso") and event.key in ["control", "ctrl+shift"]:
```